inlong-dashboard/src/ui/pages/ModuleAudit/IdModule/config.tsx

```diff
@@ -51,17 +51,20 @@ export const getFormContent = initialValues => [
     name: 'inlongGroupId',
     props: {
       dropdownMatchSelectWidth: false,
+      showSearch: true,
       options: {
         requestAuto: true,
-        requestService: {
+        requestTrigger: ['onOpen', 'onSearch'],
+        requestService: keyword => ({
```

Comment:
No `filterOption`. It's too customized
Try to hand over the filter logic to the instance:
```js
requestService: keyword => {
    const res = await request()
    return res.filter()
}
```